src/lib/Module/Components/teaserList.svelte

```diff
@@ -30,16 +14,15 @@
         <h1
           class="text-2xl font-extrabold tracking-tight text-gray-900 sm:text-3xl"
         >
-          Lesmodules
+          {$t("modules")}
         </h1>
       </div>
     </div>
 
-    <!-- Products -->
-    <section aria-labelledby="products-heading" class="mt-6">
-      <h2 id="products-heading" class="sr-only">Lesmodules</h2>
+    <section aria-labelledby="modules-heading" class="mt-6">
+      <h2 id="modules-heading" class="sr-only">{$t("modules")}</h2>
 
-      {#if modules}
+      {#if modules && modules.length > 0}
```

Comment:
You have `&& modules.length > 0` here, and `{:else}   {$t("modules-not-found")}    {/if}` down below

Reply:
Yes, it goes into the if statements if there are modules found else it will display the translation message